ghz-tool/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu
+
+RUN apt-get -q update \
+    && apt-get -q -y install wget \
+    && wget https://github.com/bojand/ghz/releases/download/v0.58.0/ghz-linux-x86_64.tar.gz \
```

Comment:
Nice! Maybe we could use the latest stable release of `ghz`, unless the API drastically changed?